pkg/aws/ec2/ec2.go

```diff
@@ -32,30 +32,52 @@ var (
 var (
 	InstanceCPUHourlyCostDesc = prometheus.NewDesc(
 		prometheus.BuildFQName(cloudcostexporter.MetricPrefix, subsystem, "instance_cpu_usd_per_core_hour"),
-		"The cpu cost a compute instance in USD/(core*h)",
-		[]string{"instance", "region", "family", "machine_type", "cluster", "price_tier"},
+		"The cpu cost a ec2 instance in USD/(core*h)",
+		[]string{"instance", "Region", "family", "machine_type", "cluster", "price_tier"},
```

Comment:
I think we must do `exported_cluster` for the cluster label. We use `cluster` otherwise at Grafana

Reply:
Ahhh, let me align this with GKE and use `cluster_name`: https://github.com/grafana/cloudcost-exporter/blob/0c38ba570a79e04c74a8283012b10b40e88c17ca/pkg/google/gke/gke.go#L45